### ISSUE TITLE
Translate everything

### ISSUE THREAD

**Issue by amosonn** [circleofcircles/circleofcircles#1]
As discussed in README.rst, since our goal is to get to as many people as possible, it's probably in our interest to have everything translated at least to English. But probably having as many languages as possible is also conducive.
I'm leaving this here so we have an issue to put related stuff in; my current proposal is *not* to translate things, at least until we've made some basic progress which shows this is actually going anywhere.

**Comment by simsim314** [comment]
I agree. Just want to say for me english is about 15% more effort than Hebrew, and writing in hebrew and then translating is about 30% extra effort in my opinion. So there is definetelly a point to write everything in english but this can have also some other consequences to other people in the project and motivation etc.

**Comment by amosonn** [comment]
That's why I want to discuss, at what point we want to do this, and how.

For me, in general expressing myself in Hebrew and in English is almost the
same amount of work; in these subjects it might be a little more work in
English, because I have less experience and also the vocabulary we've
developed so far is mostly in Hebrew, but I think that after getting used
to it it will be normal. On the other hand, I think you underestimate the
effort required for translation. However, my point was originally to
decrease the effort of the first writing, so we get as much of the content
and understanding as we can, at the price of working more afterwards.

I also assume that there are other native Hebrew speakers who we might want
to bring in, for whom the extra cost of writing in English, or perhaps even
reading in English, will be more prohibitive. That's why I suggest that we
keep the Hebrew versions even after we translate them; and that's also why
I assigned this ticket to myself, as I'm willing to translate from Hebrew
to English in such cases, and perhaps even the other way (though I assume
there would be other people up to that task as well, and perhaps it is
better that they do that and leave the English to me).

But this is assuming that we reach a conclusion, that we do indeed want to
have this whole thing in English, or in English and optionally in other
languages of source (for now mostly Hebrew), or Bi-lingual all the way, and
when. If we don't reach such an agreement, it is pointless to run around
translating random things.

My suggestion is: We want to have the whole thing in as many languages as
possible. After some point, we want to attempt to have everything in
English, and also try to keep in parallel everything in Hebrew. We should
choose some point, defined probably by time or volume, but which can be
vaguely defined: after this point, we will start translating all our pages
to English, the guideline will be to write new pages in English when
possible, and at lower priority, we will translate pages written originally
in English back to Hebrew.

But I'm willing to hear any contentions with this suggested plan.

Also, keep in mind, that we might need to handle other languages sooner
than we think. I would be very happy to share this with some of my friends
here as well, hopefully they'll want to contribute as well, and no doubt
some of them will prefer writing in German. We don't need to future-proof
against this quite yet; but it would be nice if our plan did support this
seamlessly. I assume that I will be able to review, and probably translate
to English, German texts as well, in the worst case with some help in
clarification of difficult points. If there's one thing I learned from my
translation class, is that it relies much more on mastery of the target
language than of the source one.

**Comment by simsim314** [comment]
Sometimes people join to github and help, so meybe some people will chose to translate instead of contributing. 

My main point is that we should get critical mass of issues and exercises that feel confident enough to have something even remotelly reminding some sort of school of thought, and more people will recognize the basic value of this project and be able to give fredback. 

Think of it as pilot in hebrew - and if the pikit takes of we can start from translating.

**Comment by amosonn** [comment]
So you want to wait for us to achieve some mass in Hebrew users (and not
just entries) before going through translation?

I have two arguments for early moving to English. The first is rather
personal, and that is that I would like to share this work with friends
here as well.
The second, is that the longer we wait, the more effort it is to translate
everything - as opposed to switching to writing English earlier.

Why do you want to keep it Hebrew (except for saving the extra work, in
case it doesn't pick-up or we abandon it)?

**Comment by simsim314** [comment]
I feel less comfortable with english and I understand the motiavation but I think more of my friends will fall into category of ignorance if written in english. 

It might be useful then to translate some of it to english if you wish people to aknowledge some ideas. I think there could be other reasons like we get to some middle ground like making it short and translate the short stuff to english. Don't want to waste effort now.

**Comment by amosonn** [comment]
I have another suggestion - maybe we want to have a completely separate
repo for English, like in Wiki? I would still push off opening it for now,
but I guess this makes organization easier, and we can always translate and
copy content from one to the other.
If not, then I accept your opposition, I'll leave the issue open and we can
come back to it later on.

**Comment by simsim314** [comment]
My problem with english rep is organizational i.e. making sure everything is translated. As it concerns me you should decide, I think for this stage of development this question is like where do we want to fly on a plain that we now have only wheels to.

**Comment by amosonn** [comment]
Well, as I told you, you have friends that might ignore it if it is in
English, I have friends that can't read it at all if it's not.
But OK, I'll leave this for now, we'll come back sometime later.

**Comment by simsim314** [comment]
You're always welcome to translate stuff which you feel people ypu know might benefit from. But as O said earlier taking off might mean gaining critical mass of participants some of which might take the translation task as well as it could be shortened significantly if we will decide to reorganize stuff which we call a core and decide to translate it instead of everyhing.

**Comment by amosonn** [comment]
Of course I understand the point of gaining critical mass, which is the
only way this can succeed; and that if there are enough people then some
tasks can be off-loaded. My point it that translating to English is also an
alternative means to that end. I don't think translating just the core will
help in any way. I want to translate it not just to let some people read,
but to increase the size of the group of people who can potentially jump on
the bandwagon. So putting the personal preferences aside, we can ask, what
is more likely to help this take off - writing more entries (making it more
tempting), or making the existing ones accessible to more people (so with
the same probability of a single person joining, increasing the overall
expectancy)? I don't know what is the right answer. If we all knew mostly
people who are native Hebrew speakers, then it would be clear that the
second option would be mostly theoretical until a much later stage.
However, I currently knew quite a few who aren't, so translating to English
will indeed increase the immediate audience who will be exposed to this.
And then the question is, at what point in the development does the second
option overtake the first one.

p.s. Talking about other participants doing translation is nice; do note,
that at least from my experience with open-source projects, most of the
work is still done by the original maintainers.

**Comment by simsim314** [comment]
Ok I see your point - I'm saying that for me right now hebrew 15% seems crucial to continue contributing at higher paste, but if you decide to make it in english I'll accept and try to fit in, this is why I asked about the language in the first place and this is another reason why I think switching now after making this decision is a bad idea. Think of it as reducing my salary by 15%. If you would argue for english in the first place it would be different story. And another point is that we wasting precious  editing time on stuff we kinda agreed upon early on, and at this point we like have 5 issues and it's totally not the time by any means for this disccusion. 

Another point is that people who you think fit to contribute are defenetelly welcome to join and write stuff in english as well. I'm saying for me at this stage to write in engliah looks much less atractive option.
